docs/developer/architecture.md

```diff
@@ -22,11 +22,10 @@
 ## Overview
 
 Ballista allows queries to be executed in a distributed cluster. A cluster consists of one or
-more scheduler processes and one or more executor processes. See the following sections in this document for more
-details about these components.
+more scheduler processes and one or more executor processes.
```

Comment:
Strictly speaking we only support a single scheduler at the moment. But maybe we keep it like this since I hope we can fix that soon :)